mobile_app_inventory/static/www/views/main_scan.html

```diff
@@ -22,7 +22,7 @@ <h1 class="title">{{'Inventory' | translate}} {{data.inventory.name}}</h1>
 
                 <label class="item item-input" toggle-anim="data.location">
                     <div class="form-field">{{'Current Location' | translate}}</div>
-                    <span class="form-label">{{data.location.name}}</span>
+                    <span class="form-label">{{ data.location.parent_complete_name }} / {{data.location.name}}</span>
```

Comment:
abstain on that point. For it depends of how location are named.

Pro to display parent complete name : sometime the name of the location is not sufficient to know exactly on wich location we are working. 
Pro to hide parent complete name : the display can be ugly (very big name), on small devices.
if parent complete name is very big, is the location name hidden ? if no, :+1:

Reply:
What do you suggest then ?

Reply:
maybe 
```
{{data.location.name}} ({{data.location.parent_complete_name}})
```

well location.name should be display in any case. Don't you think ?